src/events/app/src/main/resources/application-app.yml

```diff
@@ -1,3 +1,7 @@
-kafka:
-  bootstrap:
-    servers: localhost:9092
+kafka:
+  bootstrap:
+    servers: localhost:9092
+test-resources:
+ containers:
+   kafka:
+     image-name: "confluentinc/cp-kafka:7.3.3"
```

Comment:
Do we need this specific version? Can we not use Test Resources' default?

Reply:
sure we can